app/controllers/application_controller.rb

```diff
@@ -384,4 +384,12 @@ def mobile?
     client = DeviceDetector.new(request.user_agent)
     client.device_type != 'desktop'
   end
+
+  def override_ahoy_user
```

Comment:
Do we still need this? Or anything else, other than the changes in d3b6d163f1367a57a21dfcae1b7f92bf7d4a2ca2 ?

Reply:
Ah good point!